### PR TITLE
Adding classes from the internal sdk without modification

### PR DESCRIPTION
**Pull request by joannehill**
In support of CITR-1915 we are adding these classes so that we can provide users access to the data view summary information about each column. Actual support for getting this summary information will be done in a separate PR.

### CODE REVIEW

**Comment by maxhutch** [comment]
@joannehill the import paths to refer to the internal SDK.

**Comment by maxhutch** [comment]
The python2.7 test is passing but 3.x are failing.

File: citrination_client/models/columns/__init__.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from base import BaseColumn
```
Comment (mcgalcode) [inline citrination_client/models/columns/__init__.py]: These import statements will cause tests to fail in python 3.x.

I think you need to either change them to use the full path (e.g. `from citrination_client.models.columns.base import BaseColumn`), or use the relative module path syntax (e.g. `from .base import BaseColumn`)

Comment (joannehill) [reply citrination_client/models/columns/__init__.py]: Thanks, yup. Will replace with full paths

**Comment by mcgalcode** [comment]
@hutch any other comments here?

**Pull request merged**